catalog/product_info.php

```diff
@@ -206,7 +206,7 @@
     if ($Qproduct->value('products_date_available') > date('Y-m-d H:i:s')) {
 ?>
 
-    <div class="alert alert-info"><?php echo OSCOM::getDef('text_date_available', ['products_date_available' => DateTime::toLong($Qproduct->value('products_date_available'))]); ?></div>
+    <div class="alert alert-info"><?php echo sprintf(OSCOM::getDef('text_date_available'), DateTime::toLong($Qproduct->value('products_date_available'))); ?></div>
```

Comment:
Thanks for the PR! Could you actually please revert this and instead change `%s` in the language definition for `text_date_available` with `{{products_date_available}}`

Thanks! 🍺